test-apps/ui-test-app/.env

```diff
@@ -0,0 +1,31 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
@aruniverse we'll have to see how this plays with the `rush copy:config` script.  Any reason to not keep the `.env.template`? The intent is to have someone copy it and create their own `.env`.

Reply:
Also surprised you were able to even commit it :)

Reply:
Was unaware of the internal-tools copy-config script and what it does. 
imo, we should provide the `.env` file, and users will override it with their values for their imodel and etc in a `.env.local`. I can add comments to the `.env` file alerting the user to do this.

I was able to commit these changes because i updated the `.gitignore` 😅

Reply:
This is going to cause a handful of issues with our current workflow.  I do like the change but lets hold off since I want to ensure everyone doesn't accidentally now commit their .env with secrets in them.

I'm going to move all of these items back into the package.json directly rather than in a `.env`.